src/bin.js

```diff
@@ -2,4 +2,6 @@
 
 'use strict'
 
-require('./')()
+// First param is the target version
+// Second param is the target platform
+require('./')(process.argv[2], process.argv[3], function () {})
```

Comment:
Rather than passing in an empty function, why not make the callback either optional or useful and print something like `successfully installed <version>`